src/__tests__/a11yAPI.test.tsx

```diff
@@ -29,11 +28,11 @@ const waitForOptions = { timeout: 10 };
 class Button extends React.Component<any> {
   render() {
     return (
-      // $FlowFixMe - accessibilityStates removed in RN 0.62
       <TouchableOpacity
         accessibilityHint={BUTTON_HINT}
         accessibilityLabel={BUTTON_LABEL}
         accessibilityRole="button"
+        // @ts-ignore - accessibilityStates removed in RN 0.62
```

Comment:
Should we consider dropping support for that? Just curious

Reply:
maybe try accessibilityState={{ selected: true }}

Reply:
We do support `accessibilityState` (see [below](https://github.com/callstack/react-native-testing-library/pull/881/files/0ad5e48dc9d1fbaa8bb382708a93dbb7dbef68d2#diff-7a8cb5b84e76140224edf2c69459eef7246f003a6ddd3b67b03660f86dc68681R62) for instance, my question is that since RN 0.62 doesn't support `accessibilityStates` anymore, should we support it? And if yes, for how long?

Reply:
Since the oldest React-Native version mentioned in docs is `>= 0.61` and I see no clear "minimum supported React Native version" info, I'd assume people can be using RNTL in 0.61~ versions where `accessibilityStates` prop is still around. Thus I'd keep it if it does not incur big maintenance cost.